src/common/components/search/SearchItems.tsx

```diff
@@ -2,25 +2,31 @@ import React, { FC } from 'react';
 import { Avatar } from '../elements/Avatar';
 import { shortenAddress } from '@/modules/utils/string';
 import Link from 'next/link';
+import classNames from 'classnames';
 
 interface ProfileSearchItemProps {
   address: string;
   handle?: string | null;
   profileImage?: string;
   onClick?: () => void;
+  active?: boolean;
 }
 
 export const ProfileSearchItem: FC<ProfileSearchItemProps> = ({
   address,
   handle,
   profileImage,
   onClick,
+  active,
 }) => {
   return (
     <Link href={`/profiles/${address}`}>
       <a
         onClick={onClick}
-        className={`flex flex-row items-center justify-between rounded-lg p-4 hover:bg-gray-800`}
+        className={classNames(
+          `flex flex-row items-center justify-between rounded-lg p-4 hover:bg-gray-800`,
+          active && 'bg-gray-800'
```

Comment:
```suggestion
          { 'bg-gray-800': active }
```

I prefer hash options but nb.